packages/dustkit/src/apps/getProgramAppConfigUrl.ts

```diff
@@ -18,17 +18,30 @@ export async function getProgramAppConfigUrl({
   program: ProgramId;
   entity?: EntityId;
 }): Promise<string | undefined> {
-  const url = await readContract(client, {
+  let url: string | undefined;
```

Comment:
you can move this and the empty string check inside the try/catch and then you don't need a let hanging around

Reply:
updated